src/components/album/AlbumPreview.js

```diff
@@ -4,6 +4,7 @@ import { CardContent } from '@mui/material';
 const AlbumPreview = ({
   spotifyUrl,
   style = {},
+  contentLayout,
```

Comment:
To clarify my previous comment, we still may want the default value here otherwise, it'll be `undefined` the way you're calling it in `Album.js` with no contentLayout passed in (eg: `<AlbumPreview spotifyUrl={album.spotifyUrl}>`). 

My previous comment was to ensure that you override the default for `AlbumChat` so that we don't accidentally mess with how that page is rendered. 

```suggestion
  contentLayout = "column",
```

Reply:
@akosasante to be honest, I am lost a bit with the hierarchy- which component is the father and passes props to it's children. I see it as `AlbumList.js` fetches all the albums and passes them to `AlbumGrid.js` , this one  passes '**each'album** to  `Album.js` component  that displays `AlbumPreview.js` and `AlbumReviews `- the first one is basically the card /design that shows it's content- so if `Album.js `is the parent component for `AlbumPreview.js`, and we don't pass contentLayout , so we should pass it directly along with props = declare it for the first time, right? here on line 7 (img above) so that we could use it when render the page- right?
![image](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/103755090/ead74306-d2a6-4976-a7aa-96bf9705eb1f)